modules/boost.detail/1.83.0/MODULE.bazel

```diff
@@ -0,0 +1,13 @@
+module(
+    name = "boost.detail",
+    version = "1.83.0",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 108300,
+)
+
+bazel_dep(name = "rules_cc", version = "0.0.9")
```

Comment:
Can you sort the dependencies alphabetically?

Reply:
updated [here](https://github.com/bazelbuild/bazel-central-registry/pull/2580/commits/32c00526908b7cdf740c9bfdb302f80c0e46e99a)